features/unit/v2algodclient_paths.feature

```diff
@@ -32,15 +32,15 @@ Feature: Algod REST API v2
     Examples:
       |path | max| format |
       |/v2/transactions/pending?format=msgpack | 0  | msgpack |
-      |/v2/transactions/pending?max=1&format=msgpack | 1  | msgpack |
+      |/v2/transactions/pending?format=msgpack&max=1 | 1  | msgpack |
```

Comment:
There are some more in this file

Reply:
@winder I'd like to leave these particular ones as-is because `go` and `js` can only request msgpack, they cannot generate `?format=json` or `format`-omitted queries (this was requested back during design in March)

Reply:
Maybe we can have the convention be `format=msgpack` goes first. That can be the only artificial order restriction.